dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/JobType.java

```diff
@@ -275,7 +275,7 @@ public Map<String, String> getRelativeApiElements() {
               "relativePeriods", "/api/periodTypes/relativePeriodTypes",
               "validationRuleGroups", "/api/validationRuleGroups");
       case PUSH_ANALYSIS -> Map.of("pushAnalysis", "/api/pushAnalysis");
-      case TRACKER_SEARCH_OPTIMIZATION ->
```

Comment:
I don't think this case is required anymore. It previously was intended to use to populate the jobconfiguration dropdown for "which attributes to index". But since we dont have that anymore, we can remove this section